tests/src/application/validation/validators/password-validation.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { InvalidParamError } from '@/application/errors'
+import { PasswordValidation, type PasswordValidator } from '@/application/validation'
+
+const makePasswordValidatorStub = (): PasswordValidator => {
```

Comment:
We have a proper folder to group the Stubs and make test file more clean, specially if you will reutilize, check it